tests/phpunit/tests/block-supports/layout.php

```diff
@@ -173,4 +173,306 @@ function test_outer_container_not_restored_for_aligned_image_block_with_themejso
 
 		$this->assertSame( $expected, wp_restore_image_outer_container( $block_content, $block ) );
 	}
+
+	/**
+	 * Generates the CSS corresponding to the provided layout.
+	 *
+	 * @ticket 56467
+	 *
+	 * @dataProvider data_wp_get_layout_style
+	 *
+	 * @covers ::wp_get_layout_style
+	 *
+	 * @param array  $args {
+	 *       Arguments for the test function.
+	 *
+	 *      @type string  $selector                      CSS selector.
+	 *      @type array   $layout                        Layout object. The one that is passed has already checked the existence of default block layout.
+	 *      @type boolean $has_block_gap_support         Whether the theme has support for the block gap.
+	 *      @type string  $gap_value                     The block gap value to apply.
+	 *      @type boolean $should_skip_gap_serialization Whether to skip applying the user-defined value set in the editor.
+	 *      @type string  $fallback_gap_value            The block gap value to apply.
+	 *      @type array   $block_spacing                 Custom spacing set on the block.
+	 * }
+	 * @param string $expected_output The expected output.
+	 */
+	function test_wp_get_layout_style( $args, $expected_output ) {
+		$layout_styles = wp_get_layout_style( $args['selector'], $args['layout'], $args['has_block_gap_support'], $args['gap_value'], $args['should_skip_gap_serialization'], $args['fallback_gap_value'], $args['block_spacing'] );
```

Comment:
```suggestion
		$layout_styles = wp_get_layout_style(
			$args['selector'],
			$args['layout'],
			$args['has_block_gap_support'],
			$args['gap_value'],
			$args['should_skip_gap_serialization'],
			$args['fallback_gap_value'],
			$args['block_spacing']
		);
```
Just to shorten the line length, which was quite long.